docs/events/background_events.md

```diff
@@ -25,6 +25,10 @@ In the following example, the `my_task` event handler is decorated with
 long as certain conditions are met. While it is running, the UI remains
 interactive and continues to process events normally.
 
+```md alert info
+# Background events are similar to Task Queues in [Celery](https://www.fullstackpython.com/celery.html) allowing asynchronous events.
```

Comment:
I don't think this is true. Background events can be used to start task queues etc. but we don't offer all the features of a task queue.

Reply:
rephrased